### PR TITLE
[KO]Update osu!mania Ranking Criteria

### PR DESCRIPTION
**Pull request by Garalulu**
## Description
Korean translation of osu!mania Ranking Criteria
## Status
Finished, waiting for others opinion
## Changelog
07/25 changed some awkward phrases, unified the usage of words for consistency // Garalulu
07/26 deleted unnecessary word and minor changes // Riana, arpia97
07/30 added osu!general RC and minor changes // Riana

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!mania/ko.md [hunk 1]
```diff
@@ -1,215 +1,208 @@
...
+- **잭(Jack)**: 세 개 이상의 노트가 다른 칼럼들에 비해 빠르게 한 칼럼에 연속으로 배치된 패턴.
+- **점프(Jump)**: 동일 시간에 노트 두 개를 배치한 패턴.
+- **점프트릴(Jumptrill)**: 점프가 번갈아 가면서 나오는 트릴 패턴.
+- **미니잭(Minijack)**: 두 개의 노트가 다른 칼럼들에 비해 빠르게 한 칼럼에 연속으로 배치된 패턴.
+- **릴리즈(Release)**: 롱노트의 끝부분으로, 점수를 위해 반드시 정확한 판정에 손가락을 떼어야 합니다.
+- **롤(Roll)**: 처음부터 마지막 또는 마지막부터 처음 칼럼으로 빠르게 이어지는 일종의 스트림 패턴.
+- **쉴드(Shield)**: 롱노트 시작 전 동일 칼럼에 노트가 배치된 패턴.
+- **스플릿 점프트릴(Split-jumptrill)**: 양 손을 동시에 사용하는 점프가 이어지는 점프트릴 패턴. 양손 트릴(Two one-handed trills)이라고도 합니다.
+- **스트림(Stream)**: 미니잭을 만들지 않고 동일 간격에 지속해서 노트가 배치된 패턴. 
+- **Trill(트릴)**: 두 개의 칼럼을 번갈아 가면서 두 번 초과로 노트가 배치된 패턴.
```
Comment (Garalulu) [inline wiki/Ranking_Criteria/osu!mania/ko.md:47]: - **트릴(Trill)**: 두 개의 칼럼을 번갈아 가면서 두 번 초과로 노트가 배치된 패턴.

File: wiki/Ranking_Criteria/osu!mania/ko.md [hunk 2]
```diff
@@ -1,215 +1,208 @@
...
-- **타이밍 변경은 슬라이더 속도를 조정하여 통일된 스크롤 속도를 가져야 합니다.** 이것은 모든 타이밍 구간의 스크롤 속도중, 비트맵에서 가장 많이 사용되는 스크롤 속도로 정상화시킴으로써 이루어집니다. 이 숫자는 곡 선택에서 BPM 범위 뒤에 괄호로 표시됩니다. 예: BPM: 60-160 (130) 이 지침은 SV 효과를 생성하는 데 사용되는 슬라이더 속도 변화를 추가하는 기능에 영향을 주지 않습니다.
-- **슬라이더 속도 변화는 노래의 일반적인 강도에 따라야 합니다.**
-- **슬라이더 속도 변화는 곡의 박자 구조에 따라 끊어져야 합니다.** 곡이 올곧은 박자를 사용하고 있다면 1/4, 1/8, 1/16박자를 사용해야 하고, 변환된 박자를 사용하고 있다면 1/6, 1/12박자를 사용해야 합니다. 권장하는 박자와 다른 박자를 사용하는 경우, 실제 박자에 스냅하는 것이 항상 우선이므로 만약 곡이 1/7박자나 1/11박자를 사용하는 경우 노트가 그 박자를 따르기 위해서 기존 타이밍 라인에 스냅되지 않아도 허용이 됩니다.
-- **짧은 시간동안 키아이(kiai)를 너무 사용하면 안됩니다.** 그렇게 하는 것은 주의를 산만하게 할 수 있고 게임 플레이에 부정적인 영향을 미칠 수 있습니다.
-- **Break(휴식 시간)은 음악의 한 부분이 극도로 차분하거나 조용할 때만 사용해야 합니다.** 특히 더 높은 BPM 곡의 경우 낮은 난이도는 새로 온 플레이어가 플레이하기에 비합리적인 곡의 일부를 피하기 위해 휴식 시간을 사용할 수 있습니다.
+- **비트맵의 스크롤 속도를 변경하는 변속을 사용할 수 없습니다.** 다중 BPM의 스크롤 속도를 통일하기 위한 변속은 예외입니다.
+- **스크롤 속도 통일은 반드시 다중 BPM을 가진 비트맵에만 적용되어야 합니다.**
 
-## Difficulty-specific (각자 난이도에 따른 자세한 사항)
+#### Guidelines
```
Comment (Garalulu) [inline wiki/Ranking_Criteria/osu!mania/ko.md:114]: #### 가이드라인(Guidelines)

File: wiki/Ranking_Criteria/osu!mania/ko.md [hunk 3]
```diff
@@ -1,215 +1,208 @@
...
-- **가까운 column에서 동시에 눌린 롱 노트들은 같은 타이밍에 끊겨야 한다.** 끊어야 하는 타이밍이 여러개 존재하는 롱노트는 이 수준의 플레이어들에게 너무 어렵다.
-- **3개 또는 그 이상의 노트를 사용하는 anchor 패턴의 사용을 피하십시오.** 그것들은 플레이어가 읽기 쉬운 반면에, 더 긴 anchor는 힘들 수 있습니다.
-
-Normal이 *비트맵 세트의 가장 낮은 난이도인 경우*, 다음과 같은 추가 **rule** 을 준수해야 합니다. :
-
-- **3개보다 많은 연속적인 1/4 노트 스냅은 허용되지 않습니다.** 비트맵 세트의 가장 낮은 난이도에서 밀도가 높은 리듬은 적합하지 않습니다.
+추가적인 *7키 Hard* 가이드라인:
 
-### ![](/wiki/shared/diff/hard-m.png) Hard
+- **노트 3개 초과하는 코드 사용을 권장하지 않습니다.**  롱노트가 지속하는 경우도 하나의 노트로 계산합니다.
```
Comment (Garalulu) [inline wiki/Ranking_Criteria/osu!mania/ko.md:168]: - **노트 4개 초과하는 코드 사용을 권장하지 않습니다.**  롱노트가 지속하는 경우도 하나의 노트로 계산합니다.
not 3, it's 4

File: wiki/Ranking_Criteria/osu!mania/ko.md (hunk 2 above)
Comment (Garalulu) [reply wiki/Ranking_Criteria/osu!mania/ko.md:114]: deleted all english for Rule / Guideline

**Comment by hikizisa** [comment]
some changes I merged from ppy/osu-wiki later seems to have been mixed in this pull request

**Comment by Garalulu** [comment]
This pr is moved on #3953

**Pull request closed without merging**